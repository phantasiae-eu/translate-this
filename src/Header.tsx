import React, { ReactElement } from 'react'
import { View, Text } from 'react-native'

export default class Header extends React.Component {
    public render(): ReactElement {
        return (
            <View>
                <Text>Translate This</Text>
            </View>
        )
    }
}
