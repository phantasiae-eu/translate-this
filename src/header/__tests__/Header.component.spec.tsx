import React from 'react'
import Header from '../Header.component'

import renderer from 'react-test-renderer'

describe('Header.component', (): void => {
    it('should correctly render the header', (): void => {
        const tree = renderer.create(<Header />).toJSON()
        expect(tree).toMatchSnapshot()
    })
})
