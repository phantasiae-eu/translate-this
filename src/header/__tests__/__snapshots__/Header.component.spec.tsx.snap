// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header.component should correctly render the header 1`] = `
<View>
  <Text>
    Translate This
  </Text>
</View>
`;
